Mathlib/Order/RelSeries.lean

```diff
@@ -651,6 +651,46 @@ noncomputable def comap (p : LTSeries β) (f : α → β)
   LTSeries α := mk p.length (fun i ↦ (surjective (p i)).choose)
     (fun i j h ↦ comap (by simpa only [(surjective _).choose_spec] using p.strictMono h))
 
+/-- In ℕ, two entries in an `LTSeries` differ by at least the difference of their indices.  -/
+lemma toFun_add_sub_le_toFun_nat (p : LTSeries ℕ) (i j : Fin (p.length + 1))
```

Comment:
We often use `apply` instead of `toFun` in names

Does it make sense to have several versions of this statement?
E.g.
```
j - i ≤ p j - p i     -- what the docstring says
p i + j ≤ p j + i     -- avoids Nat subtraction
```